src/components/learner-credit-management/data/utils.js

```diff
@@ -187,3 +188,22 @@ export const orderOffers = (offers) => {
 export function formatDate(date) {
   return dayjs(date).format('MMM D, YYYY');
 }
+
+export function formatCurrency(currency) {
+  const currencyUS = Intl.NumberFormat('en-US', {
+    style: 'currency',
+    currency: 'USD',
+    minimumFractionDigits: 0,
+  });
+
+  return currencyUS.format(currency);
+}
+
+export function getEnrollmentDeadline(enrollByDate) {
```

Comment:
[inform] explanation for this logic in slack thread: https://twou.slack.com/archives/C049WNMFFRP/p1697216910565489